packages/graalvm/src/test/kotlin/elide/runtime/gvm/js/node/AbstractJsModuleTest.kt

```diff
@@ -39,20 +39,25 @@ internal abstract class AbstractJsModuleTest<T: GuestIntrinsic> : AbstractJsIntr
     // nothing
   }
 
-  protected val polyglotEngine by lazy {
-    PolyglotEngine {
-      install(JavaScript)
-      environment { configureEnvironment() }
-      vfs { configureVfs() }
-    }.acquire()
+  @OptIn(DelicateElideApi::class)
+  protected val polyglotContext by lazy {
+    engine.acquire()
   }
 
   open val supportsEsm: Boolean get() = true
   open val supportsCjs: Boolean get() = true
 
+  override fun configureEngine(config: PolyglotEngineConfiguration) {
+    config.apply {
+      install(JavaScript)
+      environment { configureEnvironment() }
+      vfs { configureVfs() }
+    }
+  }
+
```

Comment:
Moved custom configuration to this callback provided by the `AbstractDualTest` class